### PR TITLE
added margin, border, and padding to several elements

### PR DESCRIPTION
**Pull request by akatz6**

### CODE REVIEW

**Review by jameswright1902 — COMMENTED** [review]
Dope work

**Review by BeBurgosCode — COMMENTED** [review]
Looks Good!

File: style.css [hunk 1]
```diff
@@ -12,4 +13,6 @@ h1 {
 
 .textStuff {
   text-align: center;
+  margin: 10px 5px 15px 20px;
```
Comment (iAMslim) [inline style.css:16]: dont like this margin

**Review by GeecheeCipher — COMMENTED** [review]
great code.

**Comment by PaulTrusso** [comment]
great margins!

**Review by TFreeman00 — COMMENTED** [review]
Love it

**Review by iAMslim — COMMENTED** [review]
don't like margins

**Pull request merged**